api/src/test/java/org/openmrs/CohortTest.java

```diff
@@ -168,7 +168,7 @@ public void setMemberIds_shouldSupportLargeCohorts() {
         c.setMemberIds(ids);
         long endTime = System.currentTimeMillis();
         double secondsToSet = (endTime - startTime)/1000;
-        Assert.assertTrue("Setting cohort of size " + cohortSize + " took " + secondsToSet + " seconds", secondsToSet < 5);
+        Assertions.assertTrue(secondsToSet < 5, "Setting cohort of size " + cohortSize + " took " + secondsToSet + " seconds");
```

Comment:
no need to call `Assertions.assertTrue` since you already imported assertTrue in line 30